edm4eic.yaml

```
@@ -191,6 +191,15 @@ components:
       - uint32_t boundValuesSize          // size of bound values
       - std::array<double,16> transform   // row-wise 4x4 affine transform [R T; 0 1] with 3x3 rotation matrix R and translation column 3-vector T
 
+  ## An individual sample output by a HGC/CALOROC chip
+  edm4eic::CALOROCSample:
```

```suggestion
  ## An individual sample output by a HGCROC chip
  edm4eic::HGCROCSample:
```
As discussed, we don't know much about CALOROC, so this is HGCROC